### PR TITLE
Martynas-p

### PR DESCRIPTION
**Pull request by martynas3113**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,13 +23,17 @@ <h1>19 grupė</h1>
...
+            <a href="https://github.com/belauzas">(Github)</a>
+        </li>
         <!-- n -->
         <!-- o -->
         <!-- p -->
         <!-- r -->
         <li>
-            <span>Rimantas</span>
-            <a href="https://github.com/belauzas">(Github)</a>
+            <span>Martynas-P</span>
```
Comment (Vaidmine) [inline index.html]: klaida

File: index.html [hunk 2]
```diff
@@ -23,13 +23,17 @@ <h1>19 grupė</h1>
         <!-- k -->
         <!-- l -->
         <!-- m -->
+        <li>
```
Comment (Vaidmine) [inline index.html:54]: klaida

**Review by Vaidmine — DISMISSED** [review]

**Review by 2MariusG — DISMISSED** [review]

**Review by Vaidmine — APPROVED** [review]
ok

**Review by Alekslov — APPROVED** [review]

**Review by JRybelis — DISMISSED** [review]
martyna > martynas-p

**Review by LaimonasMas — APPROVED** [review]
liux

**Review by Zymantasdan — APPROVED** [review]

**Pull request merged**